internal/nametransform/diriv.go

```diff
@@ -75,16 +75,17 @@ func fdReadDirIV(fd *os.File) (iv []byte, err error) {
 // WriteDirIV - create diriv file inside "dir" (absolute ciphertext path)
 // This function is exported because it is used from pathfs_frontend, main,
 // and also the automated tests.
-func WriteDirIV(dir string) error {
+func WriteDirIV(dirfd *os.File, dir string) error {
```

Comment:
Function comment needs to be updated and dirfd explained.

Question: If dirfd != nil, can "dir" contain slashes? Should we check + panic?

Reply:
OK

It wouldn't be a problem if it contains slashes. Not sure if we should be too strict here.

Reply:
It would work because the syscall accepts it, but is it a bug in gocryptfs if that happens?